src/hooks/screen/useChatScreenHook.ts

```diff
@@ -334,8 +332,10 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
   }, [localDb, otherListener[`chat_${selectedChannel?.id}`], selectedChannel]);
 
   React.useEffect(() => {
+    setCurrentChatScreen(type === 'SIGNED' ? 'SignedChatScreen' : 'AnonymousChatScreen');
     return () => {
       setChats([]);
+      setCurrentChatScreen(null);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This `useEffect` hook is being used to set the `currentChatScreen` state when the component mounts and reset it when the component unmounts. However, it's not dependent on any props or state, which means it will only run once when the component mounts and never again. If you want this effect to run whenever the `type` prop changes, you should include `type` in the dependency array.
```diff
-  }, []);
+  }, [type]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->